src/utils/tree_node.C

```diff
@@ -457,25 +460,30 @@ unsigned int TreeNode<N>::n_active_bins() const
 
 
 template <unsigned int N>
-const Elem* TreeNode<N>::find_element(const Point& p, const std::set<subdomain_id_type> *allowed_subdomains) const
+const Elem*
+TreeNode<N>::find_element
+(const Point& p,
+ const std::set<subdomain_id_type> *allowed_subdomains,
+ Real relative_tol) const
 {
   if (this->active())
     {
       // Only check our children if the point is in our bounding box
       // or if the node contains infinite elements
-      if (this->bounds_point(p) || this->contains_ifems)
+      if (this->bounds_point(p, relative_tol) || this->contains_ifems)
```

Comment:
This check can't be here.

It's a bit complicated to explain (I tried to explain in my email)....

`find_element_in_children` is expecting to be able to _actually_ search a bounding box if it calls `find_element`... regardless of what its bounding box says.

This call right here: https://github.com/libMesh/libmesh/pull/451/files?diff=split#diff-31ecb7de88a89c206400bd14cf60de8fR534

If this condition is left here then that call doesn't work properly (it will never find anything... because the only way we were looping over those children is if they _failed_ the bounding box check in the first place)